docs/ReleaseHowTo.md

```diff
@@ -0,0 +1,42 @@
+### How to release a new version of harvester CSI driver
+
+#### 1. Tag harvester CSI driver if needed. Sometimes, we could skip this part if only the support version was update on Rancher
+
+#### 2. Bump new version for Rancher apps
+
+  2.1. For Rancher apps, you need to update the following files:
```

Comment:
can we please clarify that this change needs to be made in the `rancher/charts` repo. As a first time user even I was a bit confused how changes in harvester-csi-driver repo were being replicated to the `rancher/charts` repo.

Reply:
Updated! Please help to recheck!